components/Slider/src/MDCSlider.h

```diff
@@ -152,6 +161,15 @@ IB_DESIGNABLE
  */
 @property(nonatomic, assign, getter=isThumbHollowAtStart) BOOL thumbHollowAtStart;
 
+/**
+ * A default color scheme for sliders displayed on light backgrounds. This color scheme is applied
+ * by default to new MDCSlider instances.
+ */
++ (nonnull MDCBasicColorScheme *)defaultSliderLightColorScheme;
```

Comment:
Can we move these into the color themer so the base component doesn't depend on the NDCBasicColorScheme?